config/vufind/Demo.ini

```diff
@@ -91,8 +91,9 @@ services[] = 'custom'
 ; with locations of "foo" and "bar" respectively. For the second item the availability will be
 ; returned as an instance of AvailabilityStatus and with the extra status information "{"extraInfo": true}";
 ; All other details will be randomized.
-;[StaticHoldings]
+[StaticHoldings]
 ;1234 = '[{"location": "foo"}, {"location": "bar", "use_status_class": true, "extraStatusInformation": {"extraInfo": true}}]'
+testsample1 = "[{\"barcode\":\"12345678\",\"availability\":0,\"status\":\"Foo\",\"location\":\"Test Location\",\"locationhref\":false,\"reserve\":\"N\",\"callnumber\":\"Test Call Number\",\"duedate\":\"\",\"is_holdable\":true,\"addLink\":true,\"addStorageRetrievalRequestLink\":\"check\",\"addILLRequestLink\":\"check\",\"__electronic__\":false},{\"barcode\":\"12345678\",\"availability\":0,\"status\":\"Foo\",\"location\":\"main\",\"locationhref\":false,\"reserve\":\"N\",\"callnumber\":\"Test Call Number\",\"duedate\":\"\",\"is_holdable\":true,\"addLink\":true,\"addStorageRetrievalRequestLink\":\"check\",\"addILLRequestLink\":\"check\",\"__electronic__\":false},{\"barcode\":\"12345678\",\"availability\":2,\"status\":\"Check with Staff\",\"location\":\"main\",\"locationhref\":false,\"reserve\":\"N\",\"callnumber\":\"Test Call Number\",\"duedate\":\"\",\"is_holdable\":true,\"addLink\":true,\"addStorageRetrievalRequestLink\":\"check\",\"addILLRequestLink\":\"check\",\"__electronic__\":false}]"
```

Comment:
This also looks like an accidental commit...

Reply:
I'm very sorry, this wasn't intended to be comitted. I reverted it.